src/gwc/src/main/java/org/geoserver/gwc/layer/GeoServerTileLayer.java

```diff
@@ -91,12 +49,26 @@
 import org.geowebcache.util.GWCVars;
 import org.geowebcache.util.ServletUtils;
 import org.opengis.referencing.crs.CoordinateReferenceSystem;
-
-import com.google.common.base.Throwables;
-import com.vividsolutions.jts.geom.Envelope;
-import com.vividsolutions.jts.geom.Geometry;
 import org.vfny.geoserver.util.ResponseUtils;
 
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServletRequest;
+import java.awt.*;
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.*;
```

Comment:
how many of these import changes are new, and how many are different config between Eclipse and InteliJ. I am not used to seeing bulk "import java.util.*" imports even though that is valid.